coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+forge install
```

Comment:
IMO bash scripts are often hard to read/maintain, especially if you haven't looked at them for a few months. But they are very convenient and the workaround here is nice. However, let's make sure to document what's going on here with code comments—both why this script is needed, and what it's doing / how it works